build/BUILD

```diff
@@ -2,6 +2,9 @@ package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_bundle", "container_image")
 load("@io_kubernetes_build//defs:build.bzl", "release_filegroup")
+load(":code_generation_test.bzl", "code_generation_test_suite")
+
+code_generation_test_suite()
```

Comment:
I would recommend formatting this as:

```python
code_generation_test_suite(name = "code_generation_tests")
```

to help with autocomplete (and then piping this into the macro)

Reply:
good suggestion, done